packages/url_launcher/url_launcher/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 6.0.0-nullsafety.6
+
+* Fixes for Dart plugin registry.
```

Comment:
While we should fix the typos, url_launcher's desktop implementations don't need a Dart registry, so this description doesn't really make sense.